packages/manager/apps/cloud/package.json

```
@@ -22,6 +22,7 @@
     "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-cloud' --include-dependencies -- npm run dev:watch --if-present"
   },
   "dependencies": {
+    "@ovh-ux/manager-account-migration": "^0.0.1",
```

```suggestion
    "@ovh-ux/manager-account-migration": "^0.0.0 || ^1.0.0",
```
Why `0.0.1`, the package has no release yet

---

This is done. Thanks